lib/axon/losses.ex

```diff
@@ -1024,4 +1024,170 @@ defmodule Axon.Losses do
       end
     )
   end
+
+  @doc """
+  WIP
+  y_pred should be log_softmax output of shape (B, T, D),
+  y_true should be of shape (B, S),
+  where
+  B - batch size,
+  T - sequence length,
+  D - dictionary size, including blank
+  S - maximum target length
+  y_true nas to be preencoded, its first and last characters should be
+  blank, each initial character should be separated with blanks,
+  double blanks mark the end of target sequence.
+  """
+  defn ctcloss(y_true, y_pred, opts \\ []) do
```

Comment:
To match the convention in this module, let's call it `connectionist_temporal_classification`. I know it's long, but all of the other implementations use the long form as well.

Reply:
If you say so. But it's ctc_loss and CTCLoss in tf and torch.